go/vt/sqlparser/sql.y

```diff
@@ -3903,6 +3903,7 @@ table_references:
 table_reference:
   table_factor
 | join_table
+| table_function
```

Comment:
Probably better expressed as a type of table factor, so you can alias it

Reply:
Thanks; I went ahead and moved table_function to be a table_factor. There's more work needed to enable aliases for table functions, but I'm deprioritizing that in favor of getting the initial support for table functions out faster. Happy to return to enable aliases of course.